ci-operator/config/openshift/network-metrics-daemon/openshift-network-metrics-daemon-master.yaml

```diff
@@ -40,6 +40,10 @@ tests:
   commands: make unittests
   container:
     from: src
+- as: e2e-aws
```

Comment:
Hi no need to add a lane that will not run any tests.

can you please check the e2e test below call it `e2e-gcp` instead of just `e2e` and change the workflow to openshift-e2e-gcp

example:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml#L50

Reply:
I've made the changes you recommended. How does it look now? Thank you. :)